core/server/api/canary/members.js

```diff
@@ -28,8 +29,14 @@ const members = {
         ],
         validation: {},
```

Comment:
@allouis is there a reason why there's no validation at all? thought at least email could go through a basic check like we do for invitations - https://github.com/TryGhost/Ghost/blob/7b761a8/core/server/api/canary/utils/validators/input/invitations.js#L34

Reply:
The idea is that this controller is just proxying to a service, so unsure whether that validation should be handled in the service! Gonna leave it out of this PR for now, but something we can look at during cleanup